rnd/autogpt_server/autogpt_server/blocks/__init__.py

```diff
@@ -1,10 +1,11 @@
 from autogpt_server.blocks import sample, reddit, text, ai
 from autogpt_server.blocks.wikipedia import GetWikipediaSummary
+from autogpt_server.blocks.discord import DiscordSendMessage
 from autogpt_server.data.block import Block
 
 AVAILABLE_BLOCKS = {
     block.id: block
     for block in [v() for v in Block.__subclasses__()]
 }
 
-__all__ = ["ai", "sample", "reddit", "text", "AVAILABLE_BLOCKS", "GetWikipediaSummary"]
+__all__ = ["ai", "sample", "reddit", "text", "AVAILABLE_BLOCKS", "GetWikipediaSummary", "DiscordSendMessage"]
```

Comment:
Then add discord to this line too

Reply:
Sorry I missed this on you Wikipedia PR, you need to do the same for wikipedia

Reply:
@Swiftyos think i fixed it now, i tested and it still works